news.json

```diff
@@ -247,6 +247,7 @@
           "Pryser",
           "Libell"
         ],
+        "occupation": "Nyhetsredaktør Juridika",
```

Comment:
Litt usikker på om vi skal lagre dette med stor bokstav. For vi må uansett transformere det i frontend. Men da går det ikke å faktisk _ha_ stor bokstav, hvis tittelen krever det.

Det blir uansett et problem om vi _vil_ vise det et _annet_ sted _med_ stor bokstav. 🤷‍♂️